src/components/settings/SettingsLMSTab/ErrorReporting/SyncHistory.jsx

```diff
@@ -2,38 +2,58 @@ import React, { useEffect, useState } from 'react';
 
 import { camelCaseObject } from '@edx/frontend-platform/utils';
 import {
-  ActionRow, AlertModal, Breadcrumb, Button, Card, Icon, Image, Skeleton, Toast, useToggle,
+  ActionRow, AlertModal, Breadcrumb, Button, Card, Hyperlink,
+  Icon, Image, Skeleton, Toast, useToggle,
 } from '@edx/paragon';
 import { CheckCircle, Error, Sync } from '@edx/paragon/icons';
 import { getStatus } from '../utils';
 import { getTimeAgo } from './utils';
 import handleErrors from '../../utils';
+import { channelMapping, formatTimestamp } from '../../../../utils';
 import ConfigError from '../../ConfigError';
 import LmsApiService from '../../../../data/services/LmsApiService';
 
 import {
   ACTIVATE_TOAST_MESSAGE, BLACKBOARD_TYPE, CANVAS_TYPE, CORNERSTONE_TYPE,
   DEGREED2_TYPE, errorToggleModalText, INACTIVATE_TOAST_MESSAGE, MOODLE_TYPE, SAP_TYPE,
 } from '../../data/constants';
-
-import { channelMapping } from '../../../../utils';
 import ErrorReportingTable from './ErrorReportingTable';
 
 const SyncHistory = () => {
+  // the simple redirect is used for going back to the lms page
   const vars = (window.location.pathname).split('lms/');
   const redirectPath = `${vars[0]}lms/`;
   const configInfo = vars[1].split('/');
   const configChannel = configInfo[0];
   const configId = configInfo[1];
 
+  // the redirect with params is used when editing an existing config
+  let fullLink = `${((window.location.href).split('lms/'))[0]}lms/?`;
```

Comment:
nit: More descriptive name would be nice in the case that other links get added down the line, maybe something like 'editConfigUrl'.